metrics/test/test_cognitive_complexity.py

```diff
@@ -81,13 +83,51 @@ def test_breaks_in_linear_flow_returns_3(self):
         ]
         self.assertCountEqual(locations, expected_locations)
 
-    def test_cognitive_complexity_returns_12(self):
+    def test_nesting_returns_5(self):
+        new_source_repository = SourceRepository(
+            Path(
+                "./metrics/test/data/test_cognitive_complexity/test_nested_controlflows"
+            )
+        )
+        cognitive_complexity = CognitiveComplexity()
+        cognitive_complexity._source_repository = new_source_repository
+        actual = 0
+        for (
+            file_path,
+            file_info,
+        ) in cognitive_complexity._source_repository.files.items():
```

Comment:
Maybe store files in a variable before `for` to shorten this line. Makes it difficult to read when it spans 3 lines.